packages/common/src/search/hubSearch.ts

```diff
@@ -15,77 +13,27 @@ import {
   hubSearchItems,
   portalSearchItems,
   portalSearchGroups,
-  portalSearchItemsFilterGroups,
-  portalSearchGroupsFilterGroups,
-} from "./_internal";
-import {
   portalSearchUsers,
-  portalSearchUsersFilterGroups,
-} from "./_internal/portalSearchUsers";
+} from "./_internal";
 
 /**
- * Main Search function for ArcGIS Hub
+ * @private
+ * TEMPORARY - this function only exists to allow incremental
```

Comment:
would be good to change this to "DEPRECATED" next time you are in this code, b/c that's how I plan to search for things to cull in the Great Breaking Change to come